src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php

```diff
@@ -50,11 +50,17 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             return;
         }
 
+        $csrfTokenId = $options['csrf_token_id']
+            ?: $this->defaultTokenId[$builder->getType()->getInnerType()::class]
```

Comment:
Can we get back to using `if` or at least use parentheses to clarify which part is executed in which order? I guess I am not the only one who struggles to quickly grasp how chained ternary and null-coalesce operators are evaluated.

Reply:
from left to right as written, nothing fancy here, that's not ternary ops